lib/stanford-mods/coordinate.rb

```diff
@@ -65,11 +65,14 @@ def cleaner_coordinate(val)
       # @param [String] point coordinate point in degrees notation
       # @return [Float] converted value in decimal notation
       def coord_to_decimal(point)
-        regex = /(?<dir>[NESW])\s*(?<deg>\d+)[°⁰º](?:(?<min>\d+)[ʹ'])?(?:(?<sec>\d+)[ʺ"])?/
+        regex = Regexp.union(
+          /(?<dir>[NESW])\s*(?<deg>\d+)[°⁰º](?:(?<min>\d+)[ʹ'])?(?:(?<sec>\d+)[ʺ"])?/,
+          /^\s*(?<dir>[NESW])\s*(?<deg>[\d.]+)\s*$/
+        )
```

Comment:
i think this allows things like `N 78.6.2.1.4` and `S 69.` which i don't think are valid, although idk if that matters. `^\s*(?<dir>[NESW])\s*(?<deg>\d+(?:[.]\d+)?)\s*$` would reject those cases but accept the values in the test.